processor/cumulativetodeltaprocessor/processor.go

```diff
@@ -18,42 +18,27 @@ import (
 	"context"
 	"math"
 
-	"go.opentelemetry.io/collector/featuregate"
 	"go.opentelemetry.io/collector/pdata/pmetric"
 	"go.uber.org/zap"
 
 	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/filter/filterset"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/cumulativetodeltaprocessor/internal/tracking"
 )
 
-const enableHistogramSupportGateID = "processor.cumulativetodeltaprocessor.EnableHistogramSupport"
-
-func init() {
-	featuregate.GetRegistry().MustRegisterID(
-		enableHistogramSupportGateID,
-		featuregate.StageBeta,
```

Comment:
Move it to StageStable. cc @MovieStoreGuy

Reply:
The stable, will mark as true and will not let user change it to false, and log an error if they try instead of failing

Reply:
Sorry, late to this. 

The idea being with moving the gate to stable means that if any user attempts to modify the gate, they are given an error. Using the zpages, there should be a link to the issue that they can follow up on if there is any concerns about removing the gate or trying to modify it.

Reply:
@bogdandrutu @MovieStoreGuy feature gate updated.  I'd like to get this into the next release if possible so I can remove it for 0.68.0